commands/publish/index.js

```diff
@@ -595,11 +612,16 @@ class PublishCommand extends Command {
         tracker.completeWork(1);
 
         this.execScript(pkg, "postpublish");
+        this.runPackageLifecycle(pkg, "postpublish");
```

Comment:
This should be returned (just like `this.runPrepublishScripts`) so we don't drop any rejections.

Reply:
Would that cause a failing `postpublish` to skip the remaining updates? My concern is that since `postpublish` executes after the package is published, it might lead to corrupted states where some of the packages have been published but some have not.

Reply:
Well, I mean, an uncaught rejection is going to halt then entire process in the next major version of Node, so I don't see how we avoid this, unless we wait to run all the `postpublish` lifecycles after _all_ the publishes have succeeded.

And even then, it's not guaranteed. I guess I'm willing to let a vanishingly-rare `postpublish` break the build if someone is foolish enough to write a `postpublish` script that exits non-zero? We've done the best we can, we can't account for everything.

Reply:
Oh, and also, we already catch and log lifecycle errors in [`runPackageLifecycle`](https://github.com/lerna/lerna/blob/45efa2404933e890e5561ac4c3e0dd0c4c08cb04/commands/publish/index.js#L426-L432), so it's basically just good hygiene to return it.